src/main/java/redis/clients/jedis/ConnectionFactory.java

```diff
@@ -74,21 +66,9 @@ public PooledObject<Connection> makeObject() throws Exception {
     Connection jedis = null;
     try {
       jedis = new Connection(jedisSocketFactory, clientConfig);
-      jedis.connect();
```

Comment:
In the previous line of `new Connection`, connect will be called actively, so delete this line.